lib/config_finder.js

```diff
@@ -0,0 +1,30 @@
+"use strict";
+
+const
+    CONFIG_FILES = require("eslint/lib/config/config-file").CONFIG_FILES
```

Comment:
:lipstick: not sure what project style is, but elsewhere you put the first value on the line with `const`.